config/sets/contao/contao-50.php

```diff
@@ -4,8 +4,21 @@
 
 use Rector\Config\RectorConfig;
 use Rector\Renaming\Rector\FuncCall\RenameFunctionRector;
+use Rector\Renaming\Rector\Name\RenameClassRector;
 
 return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(RenameClassRector::class, [
+        \Contao\Template::class => \Contao\CoreBundle\Twig\FragmentTemplate::class,
```

Comment:
Actually unsure if we'd want to do this in the contao-50 migration as people don't need to rewrite everything to twig yet.
```suggestion
```

It may break lots of content elements / modules that already added the 'template' parameter